packages/moonstone/Divider/Divider.js

```diff
@@ -39,6 +40,7 @@ import css from './Divider.module.less';
  * @memberof moonstone/Divider
  * @ui
  * @public
+ * @deprecated since 2.6.0. Will be moved to {@link moonstone/Heading} in 3.0.0
```

Comment:
Since `Heading` doesn't exist here, we'll have to just reference it statically without the link. I'll update it here.